Dockerfile

```diff
@@ -18,18 +18,14 @@ VOLUME /config
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-# Copy build scripts
-COPY virtualization/Docker/ virtualization/Docker/
+# Copy source
+COPY . .
```

Comment:
The problem with this approach is that people will have to do a full download each time any file has changed as it invalidates all following layers.

Previously, the requirements layout could be reused if only the Python code had changed.